packages/app-project/src/components/ProjectHeader/ProjectHeaderContainer.js

```diff
@@ -9,22 +9,15 @@ import getNavLinks from './helpers/getNavLinks'
 function storeMapper (stores) {
   return {
     isLoggedIn: stores.store.user.isLoggedIn,
-    project: stores.store.project
+    projectName: stores.store.project['display_name']
```

Comment:
Out of interest, why this syntax and not `stores.store.project.display_name`?

Reply:
The linter enforces `camelCase`